components/script/script_thread.rs

```diff
@@ -485,6 +485,12 @@ impl Documents {
     }
 }
 
+impl Default for Documents {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
Let's focus on this issue for this PR. Can you move the contents of the `fn new()` method into `fn default()` and then remove `fn new()` altogether. This will require modifying all of the places that called `Documents::new()`.

Reply:
Okay, let met get on it.